docs/source/Objective_Functions/Objective_Function.md

```diff
@@ -1,25 +1,248 @@
-# Creating an Objective Function or Local Optimization Function for TQ42
+# Creating an Objective Function and Local Optimization Function for TQ42
 
 An objective function serves as a foundational component for optimization algorithms, guiding the computation towards an optimal solution. It is a precise mathematical expression defining the criteria for the "best" outcome, which could include minimizing costs, maximizing revenue, finding the most efficient route, or other objectives. In cases of multi-objective optimization([1]), additional functions are added to accommodate several objectives simultaneously, balancing trade-offs and providing a set of optimal solutions known as the Pareto front.([2]) This approach allows the algorithm to evaluate potential solutions against a spectrum of criteria and converge on the one that best satisfies the composite set of predefined goals.
 
-The TetraOpt algorithm can optionally accept a local optimization function to refine optimization estimates. This is 
+The TetraOpt algorithm can also optionally accept a local optimization function to refine optimization estimates. This is 
 equivalent to running the TetraOpt optimization by itself, but following up each iteration with a second optimization 
 algorithm. 
 
 [1]: https://www.egr.msu.edu/~kdeb/papers/k2011003.pdf
 [2]: https://en.wikipedia.org/wiki/Pareto_front
 
-### Objective Function Format
-A TQ42 optimization algorithm must be provided with one or more real-valued functions to be optimized. The objective function(s) must be provided in the format of an endpoint accessible through https.
+## Objective Function and Local Optimization Function Format
+A TQ42 optimization algorithm must be provided with one or more real-valued functions to be optimized. The objective and local optimization function must be provided in the format of:
+1. A communication channel
+2. An https endpoint
 
+
+An example of TetraOpt parameters using the `communication channel`. Notice the
+`objective_function_channel_id` and `local_optimizer_channel_id parameter`:
+
+```
+objective_func_channel = await Channel.create(client=client)
+local_opt_channel = await Channel.create(client=client)
+
+tetra_opt_parameters = {
+    "dimensionality": 2,
+    "iteration_number": 2,
+    "maximal_rank": 4,
+    "points_number": 1,
+    "quantization": False,
+    "tolerance": 0.0010000000474974513,
+    "lower_limits": [0, 0],
+    "upper_limits": [9, 9],
+    "grid": [10, 10],
+     "objective_function_channel_id": objective_func_channel
+     #local_optimizer_channel_id parameter is optional
+    "local_optimizer_channel_id": local_opt_channel
```

Comment:
this is not giving you an id but an instance of a channel... you need to do `local_opt_channel.id`